experimental/OrthogonalDiscriminants/src/data.jl

```diff
@@ -69,7 +69,7 @@ julia> orthogonal_discriminant(t2[4])
 """
 function orthogonal_discriminant(chi::Oscar.GAPGroupClassFunction)
   tbl = parent(chi)
-  pos = findfirst(x -> x === chi, tbl)
+  pos = findfirst(===(chi), tbl)
```

Comment:
```suggestion
  pos = findfirst(x -> x === chi, tbl)
```
unary `===` does not exist